js/pages/concept-sets/concept-manager.js

```diff
@@ -349,6 +340,13 @@ define([
 			super.onPageCreated();
 		}
 
+		onRouterParamsChanged({ conceptId }) {
+			if (this.model.currentConceptMode() == 'recordcounts') {
+				this.loadRecordCounts();
+			}
+			this.loadConcept(value);
```

Comment:
Where the `value` variable comes from?

![image](https://user-images.githubusercontent.com/10010071/49018497-99d42280-f159-11e8-9774-c4219cf7804b.png)